lib/Expr/Parser.cpp

```diff
@@ -491,10 +494,6 @@ DeclResult ParserImpl::ParseArrayDecl() {
       Error("constant arrays must be completely specified.");
       Values.clear();
     }
-
-    for (unsigned i = 0; i != Size.get(); ++i) {
```

Comment:
Well, it was dead code, so I simply decided to remove it.
I will put it back as a TODO comment.